dns/zone-configs/k8s.io.yaml

```diff
@@ -0,0 +1,216 @@
+---
+# The top-level k8s.io zone
+'':
+  - type: A
+    value: 23.236.58.218 # Our vanity redirector, hosted in GKE
+  - type: MX
+    values:
+    - exchange: aspmx.l.google.com.
+      preference: 1
+    - exchange: alt1.aspmx.l.google.com.
+      preference: 5
+    - exchange: alt2.aspmx.l.google.com.
+      preference: 5
+    - exchange: alt3.aspmx.l.google.com.
+      preference: 10
+    - exchange: alt4.aspmx.l.google.com.
+      preference: 10
+  - type: TXT
+    values:
+    - google-site-verification=RJbZ_ganmSWvslSKOBG-QHv62XTjJZcigpWIFttStFs
+    - v=spf1 include:_spf.google.com ~all
+www:
+  type: CNAME
+  value: k8s.io.
+_acme-challenge:
+  type: TXT
+  value: 7h7C4eZsMVvJ08aJmb9gIRvu1lHFCHH6_qDDUtX3Li4
+
+# Our vanity redirector.  This is not just 'k8s.io', on the off chance this
+# becomes different from the main record. (@thockin)
+redirect:
+  type: A
+  value: 23.236.58.218
+
+# Main docs site is a redirect (@chenopis)
+docs:
+  type: CNAME
+  value: redirect.k8s.io.
+# All docs subdomains get sent to netlify (@chenopis)
+'*.docs':
+  type: CNAME
+  value: netlifyglobalcdn.com.
+
+# Not sure what these control, no prior notes.
+_github-challenge-kubernetes:
+  type: TXT
+  value: 8d02d39186
+_github-challenge-kubernetes-client:
+  type: TXT
+  value: 8135eb1976
+_github-challenge-kubernetes-csi:
+  type: TXT
+  value: 5aa2511ab7
+_github-challenge-kubernetes-incubator:
+  type: TXT
+  value: cc457b8e52
+_github-challenge-kubernetes-retired:
+  type: TXT
+  value: b36f0d3bca
+_github-challenge-kubernetes-sigs:
+  type: TXT
+  value: c576f646fc
+
+# Download sites
+apt:
+  type: CNAME
+  value: redirect.k8s.io.
+yum:
+  type: CNAME
+  value: redirect.k8s.io.
+dl:
+  type: CNAME
+  value: redirect.k8s.io.
+rel:
+  type: CNAME
+  value: redirect.k8s.io.
+releases:
+  type: CNAME
+  value: redirect.k8s.io.
+
+# Vanity CNAMEs.
+blog:
+  type: CNAME
+  value: redirect.k8s.io.
+changelog:
+  type: CNAME
+  value: redirect.k8s.io.
+# TODO: Where does this live and who owns it?
+cherrypick:
+  type: A
+  value: 35.184.147.212
+ci-test:
+  type: CNAME
+  value: redirect.k8s.io.
+code:
+  type: CNAME
+  value: redirect.k8s.io.
+contributor:
+  type: CNAME
+  value: kubernetes-contributor.netlify.com.
+cs:
+  type: A
+  value: 147.75.97.58
+devstats:
+  type: CNAME
+  value: k8s.devstats.cncf.io.
+# TODO: jorge and paris and which sig?
```

Comment:
devstats and discourse are both sig-contributor-experience

Reply:
ACK